hail/python/hailtop/hailctl/batch/initialize.py

```diff
@@ -0,0 +1,258 @@
+from typing import List, Optional, Tuple
+import typer
+
+from typer import Abort, Exit
+from rich.prompt import Confirm, IntPrompt, Prompt
+
+from hailtop.config import ConfigVariable
+
+
+async def setup_existing_remote_tmpdir(service_account: str, verbose: bool) -> Tuple[Optional[str], str, bool]:
+    from hailtop.aiogoogle import GoogleStorageAsyncFS  # pylint: disable=import-outside-toplevel
+
+    from .utils import (
+        InsufficientPermissions,
+        get_gcp_bucket_information,
+        grant_service_account_bucket_access_with_role,
+    )  # pylint: disable=import-outside-toplevel
+
+    warnings = False
+
+    remote_tmpdir = Prompt.ask('Enter a path to an existing remote temporary directory (ex: gs://my-bucket/batch/tmp)')
+
+    bucket, _ = GoogleStorageAsyncFS.get_bucket_and_name(remote_tmpdir)
+
+    try:
+        bucket_info = await get_gcp_bucket_information(bucket, verbose)
+    except InsufficientPermissions as e:
+        typer.secho(e.message, fg=typer.colors.RED)
+        raise Abort()
+
+    location = bucket_info['location'].lower()
+
+    if bucket_info['locationType'] != 'region':
+        typer.secho(f'WARNING: remote temporary directory {remote_tmpdir} is multi-regional. Using this bucket with the Batch Service will incur addtional ingress and egress fees.',
```

Comment:
You mention ingress fees in a few spots. AFAIK, there are no ingress fees for any network traffic in GCP. What's your thinking on mentioning ingress here?

Reply:
Aren't there inter-region network fees for reading data from us-east bucket into us-central?

Reply:
I thought those fees were interpreted as egress out of `us-east`, not sure though.